### ISSUE TITLE
Roll out BlockChain in the house. 

### ISSUE THREAD

**Issue by CCOSTAN** [CCOSTAN/Home-AssistantConfig#390]
<img width="544" alt="bear stones" src="https://user-images.githubusercontent.com/2160436/41924828-fff2deba-7938-11e8-8b29-37d74201b442.png">

**Comment by jtmoore81** [comment]
I thought about implementing this as well but was thinking about hosting it myself.

**Comment by CCOSTAN** [comment]
@jtmoore81 Yeah .. I'm open to different ideas.. I just saw the aws as a fast way to get something implemented.

**Comment by jtmoore81** [comment]
I initially came across [and](https://www.hackster.io/matt-farley/world-s-smartest-home-feat-blockchain-chores-kids-crypto-025e83#toc-blockchain-3) liked the idea and started reading different articles and trying to figure out what would work best in my system.

**Comment by CCOSTAN** [comment]
@jtmoore81 Totally the same here.   Keep me posted on anything you figure out. :)  My notes will be here as well.

**Comment by CCOSTAN** [comment]
![bear stones v2](https://user-images.githubusercontent.com/2160436/41990655-17454444-7a11-11e8-87c4-5fe21163d24f.png)

**Comment by CCOSTAN** [comment]
https://aws.amazon.com/blogs/aws/new-amazon-managed-blockchain-create-manage-scalable-blockchain-networks/

**Comment by CCOSTAN** [comment]
Blockchains are very hard. Better to think about just creating a token on an existing. Blockchain.
